snuba/query/parser/simple_arithm_expressions.py

```diff
@@ -0,0 +1,59 @@
+from parsimonious.grammar import Grammar
```

Comment:
you do not need this file. Please remove it.

Reply:
Removed.